appsec-rules/crowdsecurity/appsec-generic-test.yaml

```diff
@@ -0,0 +1,25 @@
+name: crowdsecurity/appsec-generic-test
+description: "AppSec heartbeat – trigger on GET /crowdsec-test-NtktlJHV4TfBSK3wvlhiOBnl`"
```

Comment:
maybe we should have a consistent naming `healthcheck` or something ? heartbeat isn't accurate I think